src/scripts/main.js

```diff
@@ -1 +1,85 @@
 'use strict';
+
+const success = (message) => {
+  const el = document.createElement('div');
+
+  el.setAttribute('data-qa', 'notification');
+
+  el.classList.add('success');
+
+  el.textContent = message;
+
+  document.body.append(el);
+};
+
+const error = (message) => {
+  const el = document.createElement('div');
+
+  el.setAttribute('data-qa', 'notification');
+
+  el.classList.add('warning');
+
+  el.textContent = message;
+
+  document.body.append(el);
+};
```

Comment:
These functions are almost the same, try to make one general function (you can just pass the class as a parameter)